recipe/activate.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex
+
+[[ "@cross_target_platform@" == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ "@cross_target_platform@" == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ "@cross_target_platform@" == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+CUDA_INCLUDE_DIRS=""
+CUDA_LINK_DIRS=""
+if [ "${CONDA_BUILD:-0}" = "1" ]; then
+    CUDA_INCLUDE_DIRS="${CUDA_INCLUDE_DIRS} -I${PREFIX}/${targetsDir}/include"
+    CUDA_INCLUDE_DIRS="${CUDA_INCLUDE_DIRS} -I${BUILD_PREFIX}/${targetsDir}/include"
+    CUDA_LINK_DIRS="${CUDA_LINK_DIRS} -L${PREFIX}/${targetsDir}/lib"
```

Comment:
Either `PREFIX` or `BUILD_PREFIX` is not needed as that is already added in the nvcc.profile ( `-I$(TOP)/include` ). @jakirkham Do you remember which one `TOP` points too?

Reply:
```
TOP              = $(_HERE_)/../$(_TARGET_DIR_)
```
This would become `$BUILD_PREFIX/$(_TARGET_DIR_)`. I'm not sure whether `$(_TARGET_DIR_)` is equivalent to the `$targetsDir` defined here.

Reply:
Yes, `$(_TARGET_DIR_)` will match `targetsDir`